tezos/messages/src/protocol/proto_006/mod.rs

```diff
@@ -2,4 +2,4 @@
 // SPDX-License-Identifier: MIT
 pub mod constants;
 
-pub const PROTOCOL_HASH: &str = "PsBabyM1eUXZseaJdmXFApDSBqj8YBfwELoxZHHW77EMcAbbwAS";
+pub const PROTOCOL_HASH: &str = "PsCARTHAGazKbHtnKfLzQg3kms52kSRpgnDY982a9oYsSXRLQEb";
```

Comment:
@Kyras 
this is fixed, in another PR: https://github.com/simplestaking/tezedge/pull/180
pls. close this one